WebApp/ISIS/autoreduce_webapp/static/javascript/run_variables.js

```diff
@@ -114,6 +114,13 @@
         var validateText = function validateText(){
             validateNotEmpty.call(this);
         };
+        var validateDescriptionText = function validateDescriptionText(){
+            var max_length = 50;
+            if($(this).val().length > max_length) {
```

Comment:
A detail but still, can '>' be changed to '>=' to be consistant with your error message, i.e. at the moment I can submit a string of lenght 50 OK